bin/nest_rmu_linear.csh

```diff
@@ -0,0 +1,165 @@
+#!/bin/csh
+# Create rmu_nest files with variant e-folding time 
+set echo
+set time = 1
+#
+
+setenv TOOLS /home/sm_alfal/sea/TOPAZ5/NERSC-HYCOM-CICE/hycom/hycom_ALL/hycom_2.2.72_ALL
+setenv TOOLS ~/HYCOM-tools
+set path = ( ${path} ${TOOLS}/bin )
+C
+C --- Generate HYCOM nest rmu e-folding time, nest_rmu.a
+C --- Script has idm and jdm values hardwired.
+C --- S,E,N boundaries: 20 grid pts with 3-15 day e-folding time
+
+C --- set copy command
+if (-e ~${user}/bin/pget) then
+C --- remote copy
+  setenv pget ~${user}/bin/pget
+  setenv pput ~${user}/bin/pput
+else
+C --- local copy
+  setenv pget cp
+  setenv pput cp
+endif
+setenv E `echo $cwd | awk '{print substr($0,length($0)-2,length($0))}'`
+setenv X `echo ${E} | awk '{printf("%04.1f", $1*0.1)}'`
+
+setenv DS /home/sm_alfal/sea/TOPAZ5/NERSC-HYCOM-CICE/TP5a0.06/
+
+setenv R TP5a0.06
+setenv T 05
+C
+C --- E is experiment number, from EXPT.src
+C --- R is region identifier, from EXPT.src
+C --- T is topog. identifier, from EXPT.src
+C
+C --- P is primary path,
+C --- C is scratch directory,
+C --- D is permanent directory,
+C
+#source EXPT.src
+C
+setenv P relax/${E}
+setenv S ${DS}/${P}/
+setenv IDM    `grep idm ${DS}/topo/regional.grid.b | awk '{print $1}'`
+setenv JDM    `grep jdm ${DS}/topo/regional.grid.b | awk '{print $1}'`
+
+mkdir -p $S
+cd       $S
+
+C
+touch   rmu_linear fort.51 fort.51A regional.grid.a regional.grid.b
+/bin/rm rmu_linear fort.51 fort.51A regional.grid.a regional.grid.b
+C
+C --- Input.
+C
+touch fort.51 fort.51A
+if (-z fort.51) then
+  ${pget} ${DS}/topo/depth_${R}_${T}.b fort.51 &
+endif
+if (-z fort.51A) then
+  ${pget} ${DS}/topo/depth_${R}_${T}.a fort.51A &
+endif
+C
+touch regional.grid.a regional.grid.b
+if (-z regional.grid.b) then
+  ${pget} ${DS}/topo/regional.grid.b regional.grid.b &
+endif
+if (-z regional.grid.a) then
+  ${pget} ${DS}/topo/regional.grid.a regional.grid.a &
+endif
+C
+touch rmu_linear
+if (-z rmu_linear) then
+  ${pget} ${TOOLS}/relax/src/rmu_linear . &
+endif
+wait
+chmod a+rx rmu_linear
+C
+/bin/rm -f core 
+touch core
+/bin/rm -f fort.21 fort.21A
+C
+setenv FOR021A fort.21A
+setenv FOR051A fort.51A
+C
+#234567890123456789012345678901234567890123456789012345678901234567890123456789
+cat <<'E-o-D' >! fort.99
+S,N boundaries: 20 grid pts with 3-15 day e-folding time
```

Comment:
Is this the place where we put the coordinates of the relaxation zone in the model grid? Maybe a comment would be nice.

Reply:
Every relaxation zone can be split into multiple boxes on which different e-folding time can be applied. The box coordinate is determined by if,il,jf,jl. 
I have now updated the ".csh" script where I divide each boundary zone into two boxes, also more comments. By defualt it should work with TP5 grid. I have also the rmu_linear.f to hycom_All.

Reply:
@lbertino, if you ok  you can merge and close this request